db/migrate/20230919130352_create_posts.rb

```diff
@@ -0,0 +1,15 @@
+class CreatePosts < ActiveRecord::Migration[7.0]
+  def change
+    create_table :posts do |t|
+      t.integer :AuthorId
+      t.text :Title
+      t.text :Text
+      t.date :CreatedAt
+      t.date :UpdatedAt
```

Comment:
- [ ] @Kgomotso196 Great work adding the posts' table, however the t.date :UpdatedAt and t.date :CreatedAtare not necessary since their functionality is already done using the  t.timestamps(using timestamps will create updated_at and created_at columns)

Reply:
Thank you for your constructive feedback @danmainah.